frontend/src/components/VSkeleton/VGridSkeleton.vue

```diff
@@ -44,19 +44,23 @@ export default defineComponent({
     },
     numElems: {
       type: Number,
-      default: function () {
-        if (this.isForTab === "all") return 20
-        if (this.isForTab === "image") return 30
-        return 8
-      },
+      default: 0,
     },
   },
-  setup() {
+  setup(props) {
     function getRandomSize(max = 300, min = 100) {
       return Math.floor(Math.random() * (max - min) + min)
     }
 
-    return { getRandomSize }
+    // Calculate the default element count based on isForTab
+    const elementCount = computed(() => {
```

Comment:
You will need to add `computed` to the imports on line 30.

Reply:
There are already other imports from `vue`, so you should add `computed` in the bracket with other imports on line 30:
```
import { defineComponent, PropType } from "vue"
        ^
import { computed, defineComponent, PropType } from "vue"
```